mlflow/gateway/handlers.py

```diff
@@ -0,0 +1,108 @@
+from enum import Enum
```

Comment:
```suggestion
from __future__ import annotations

from enum import Enum
```

Can we use postponed evaluations to simplify type annotations?

Reply:
This might not work with pydantic.

Reply:
pydantic uses the explicit type definitions for internal logic processing. I tried removing them but then my `@validate` logic didn't work correctly.